src/crypto/CHIPCryptoPAL.cpp

```diff
@@ -657,5 +657,36 @@ CHIP_ERROR GenerateCompressedFabricId(const Crypto::P256PublicKey & root_public_
     return status;
 }
 
+/* Operational Group Key Group, Security Salt: "GroupKey v1.0" */
+static const uint8_t kGroupSecuritySalt[]        = { 0x47, 0x72, 0x6f, 0x75, 0x70, 0x4b, 0x65, 0x79, 0x20, 0x76, 0x31, 0x2e, 0x30 };
+static const uint8_t kOperationalGroupKeySalt[0] = {};
+
+/* Group Key Derivation Function, Info: "GroupKeyHash" ” */
+static const uint8_t kGroupKeyHashInfo[]  = { 0x47, 0x72, 0x6f, 0x75, 0x70, 0x4b, 0x65, 0x79, 0x48, 0x61, 0x73, 0x68 };
+static const uint8_t kGroupKeyHashSalt[0] = {};
+
+CHIP_ERROR DeriveGroupOperationalKey(const ByteSpan & epoch_key, MutableByteSpan & out_key)
```

Comment:
Please add unit tests for these.

Reply:
I have added unit test for DeriveGroupOperationalKey and DeriveGroupSessionId, as part of the CHIPCryptoPAL tests.